### PR TITLE
Fmcadc5 fix

### PR DESCRIPTION
**Pull request by mchindri**
- Not full fixed. Just small improvements.
 - There are some issues synching the DAC after "Request indirect access" (line 158 in axi_fmcadc5_sync.c)

### CODE REVIEW

**Comment by mchindri** [comment]
v2:
 - Fix conflicts

File: projects/fmcadc5/src/fmcadc5.c [hunk 1]
```diff
@@ -364,49 +364,28 @@ int main(void)
 
 #ifdef IIO_SUPPORT
 	struct iio_device *adc_dev_0_desc;
-	struct iio_device *adc_dev_1_desc;
```
Comment (buha) [inline projects/fmcadc5/src/fmcadc5.c]: Not sure I understand why axi-ad9625-1 is being removed. Is linux code also not up to date with current HDL ? 
https://github.com/analogdevicesinc/linux/blob/b39467c90e29766ecd6e08d3f6c3d7ff3c84b420/arch/microblaze/boot/dts/vc707_fmcadc5.dts#L101

**Review by amiclaus — COMMENTED** [review]
nit: https://github.com/analogdevicesinc/no-OS/pull/1184/commits/cab3e1e3470be2914e262b212299ca0f4ab3f26e and https://github.com/analogdevicesinc/no-OS/pull/1184/commits/97eb512c982ef2cb0e120eb217fd5c4a6e2f9e3e can be squashed :) (we also fit in 5 commits per PR in this manner)

**Comment by mchindri** [comment]
> nit: [cab3e1e](https://github.com/analogdevicesinc/no-OS/commit/cab3e1e3470be2914e262b212299ca0f4ab3f26e) and [97eb512](https://github.com/analogdevicesinc/no-OS/commit/97eb512c982ef2cb0e120eb217fd5c4a6e2f9e3e) can be squashed :) (we also fit in 5 commits per PR in this manner)

There are to different changes :) (one commit one change)

File: projects/fmcadc5/src/fmcadc5.c (hunk 1 above)
Comment (mchindri) [reply projects/fmcadc5/src/fmcadc5.c]: There are 2 spi chips and there is one instance for each (axi_ad9625_0_core and axi_ad9625_1_core) 
But only one is connected to to the axi_ad9625_dma controller ( https://github.com/analogdevicesinc/linux/blob/b39467c90e29766ecd6e08d3f6c3d7ff3c84b420/arch/microblaze/boot/dts/vc707_fmcadc5.dts#L69 ) to get data -> only one output channel buffer. 
See linux iio_info: [info_adc5.txt](https://github.com/analogdevicesinc/no-OS/files/7734142/info_adc5.txt)
On noos a similar link is done with the iio_axi_adc instance and if you check below the same ad9625_dmac was used for both

**Review by amiclaus — APPROVED** [review]

**Review by dbogdan — APPROVED** [review]

**Pull request merged**